src/BlazorAdmin/Shared/Spinner.razor

```diff
@@ -0,0 +1,9 @@
+﻿
+@inherits BlazorAdmin.Helpers.BlazorComponent
+
+@namespace  BlazorAdmin.Shared
+
+<div class="spinner-border" role="status">
+    <span class="sr-only">Loading...</span>
```

Comment:
Do we want to include an actual spinner gif?

Reply:
that is a bootstrap spinner. and it will display like this https://getbootstrap.com/docs/4.4/components/spinners/#border-spinner

Reply:
Oh nice. Didn't realize that existed. Thanks!